### PR TITLE
Add Dockerfiles for cann / pytorch / llm base image

### PR DESCRIPTION
**Pull request by Yikun**
This patch adds three dockerfiles:

- `ascendai/cann:910b-ubuntu2004`:
    - python3.8.11: there are some recommand versions on python version for `pytorch` and `torch_npu`, so pin to 3.8.11
    - Ascend-cann-nnae_7.0.RC1.alpha001_linux-aarch64.run
    - test: https://hub.docker.com/repository/docker/yikunkero/cann/general

- `ascendai/pytorch:910b-ubuntu2004`
    - pytorch 5913437a40a6e45ab7e164afb7c6ec930dd40b2f cherrypick 68cb854d734
    - torch_npu 81ece7b664adc28e698044d0e9091d39a1a6dfa6
    - test: https://hub.docker.com/repository/docker/yikunkero/pytorch/general

- `ascendai/llm:910b-ubuntu2004`
    - FastChat ce3311d086055d3c4d104573b64d71e67cd613b6
    - Apply `fastchat_npu.patch`

How to test it?

```
# Launch container with Ascend NPU device
docker run --rm --network host --name llm-on-ascend --device /dev/davinci6:/dev/davinci6 --device /dev/davinci_manager --device /dev/devmm_svm --device /dev/hisi_hdc -v /usr/local/dcmi:/usr/local/dcmi -v /usr/local/bin/npu-smi:/usr/local/bin/npu-smi -v /usr/local/Ascend/driver/lib64/:/usr/local/Ascend/driver/lib64/ -v /data/disk3/sibo/Projects:/opt/ -ti ascendai/llm:910b-ubuntu2004 bash

# Now, we are inside the container with NPU device

# Query Ascend NPU info
npu-smi info

# Confirmed that the model has been downloaded
ll /opt/vicuna-13b-v1.5

# Running Vicuna-13b inference on Ascend NPU
python -m fastchat.serve.cli --model-path /opt/vicuna-13b-v1.5 --device npu --temperature 1e-6
```

[asciicast](https://asciinema.org/a/fkqwiJ55hHaNF1gmAfKFvMzOR)

### CODE REVIEW

File: cann/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    libblas3 && \
+    apt remove -y cmake && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install Python 3.8.11
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-aarch64.sh -O ~/miniconda.sh && \
+    bash ~/miniconda.sh -b -p /root/miniconda && \
+    rm -f ~/miniconda.sh
+RUN /root/miniconda/bin/conda create --name torch_npu -y python=3.8.11
+ENV PATH=/root/miniconda/envs/torch_npu/bin/:$PATH
```
Comment (Yikun) [inline cann/Dockerfile:31]: It seems we can use the python here without `conda activate`, that is just install python using conda.

cc @wangxiyuan

File: cann/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:20.04
```
Comment (wangxiyuan) [inline cann/Dockerfile:1]: we'll add other OS, like openEuler later. so the file name should be changed as well?
`Dockerfile`  to `Dockerfile-ubuntu-910b`
or add more sub folder

File: llm/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+FROM yikunkero/pytorch:910b-ubuntu2004
```
Comment (wangxiyuan) [inline llm/Dockerfile:1]: yikunkero -> ascend-ai/

File: pytorch/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,31 @@
+FROM yikunkero/cann:910b-ubuntu2004
```
Comment (wangxiyuan) [inline pytorch/Dockerfile:1]: ditto

File: .github/workflows/build.yml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+          docker build -t yikunkero/cann:910b-ubuntu2004 -f ./cann/Dockerfile .
+          docker build -t yikunkero/pytorch:910b-ubuntu2004 -f ./pytorch/Dockerfile .
+          cd llm
+          docker build -t yikunkero/llm:910b-ubuntu2004 .
+ 
+      - name: Push
+        run: |
+          docker push yikunkero/cann:910b-ubuntu2004
+          docker push yikunkero/pytorch:910b-ubuntu2004
+          docker push yikunkero/llm:910b-ubuntu2004
```
Comment (wangxiyuan) [inline .github/workflows/build.yml:34]: push to ascend-ai?

**Review by wangxiyuan — APPROVED** [review]

**Pull request merged**